.github/workflows/reusable-release.yaml

```
@@ -107,7 +107,7 @@ jobs:
       # because GoReleaser Free doesn't support pushing images with the `--snapshot` flag.
       - name: Build and push
         if: ${{ inputs.goreleaser_config == 'goreleaser-canary.yml' }}
-        uses: docker/build-push-action@v6
+        uses: docker/build-push-action@b32b51a8eda65d6793cd0494a773d4f6bcef32dc # v6.11.0
```

`docker/build-push-action@v6` used version `v6.18.0` in the last workflow run:
https://github.com/aquasecurity/trivy/actions/runs/16640963671/job/47091042143#step:1:33
https://github.com/docker/build-push-action/releases/tag/v6.18.0

```suggestion
        uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83 # v6.18.0
```